dashboards/puma/main.yml

```
@@ -0,0 +1,43 @@
+metric_keys:
+  - puma_threads
+dashboard:
+  title: Puma metrics
+  description: |
+    Track Puma workers on this dashboard, 
+    see how many worker processes are running 
+    and what the used/available concurrency is for those workers. 
+  graphs:
+    - title: Threads running/max/idle
+      line_label: '%hostname% - %type%'
```

lengthy hostnames can make this less readable. In my demo dashboard I added the hostname last so the important bit (the type) is first.

```suggestion
      line_label: '%type% - %hostname%'
```